backend/web/src/frontend/src/views/header/BookmarkHeader.vue

```diff
@@ -19,33 +19,71 @@
               </v-list-item-content>
             </v-list-item>
           </v-list>
+          <template v-slot:append>
+            <div class="text-center">
+              <v-avatar>
+                <img
+                  src="https://avatars3.githubusercontent.com/u/45934117?s=460&u=32a2980278056093510eface6c7ddfca6bcd8bc9&v=4"
+                  alt="evan"
+                />
+              </v-avatar>
+            </div>
+          </template>
         </v-navigation-drawer>
-        <v-card tile width="250">
-          <v-list v-for="{ id, title, tags } in categories" :key="id" class="grow">
-            <v-list-group class="text-left mt-2" link value="true">
-              <template v-slot:activator>
-                <v-list-item-content>
-                  <v-list-item-title class="font-weight-black text-h7">{{ title }}</v-list-item-title>
-                </v-list-item-content>
-              </template>
-              <v-list-item v-for="{ id, name } in tags" :key="id" @click.prevent="fetchBookmarks">
-                <v-list-item-content>
-                  {{ name }}
-                </v-list-item-content>
-              </v-list-item>
-            </v-list-group>
-          </v-list>
-        </v-card>
+
+        <v-col>
+          <v-navigation-drawer permanent width="100%">
+            <v-text-field
+              prepend-inner-icon="mdi-magnify"
+              placeholder="Tag Search"
+              filled
+              rounded
+              dense
+              class="ma-2"
+              style="height: 5%;"
+            ></v-text-field>
+            <v-card tile width="250" height="95%">
+              <v-list v-for="{ id, title, tags } in categories" :key="id" class="grow">
+                <v-list-group class="text-left mt-2" link value="true">
+                  <template v-slot:activator>
+                    <v-list-item-content>
+                      <v-list-item-title class="font-weight-black text-h7">{{ title }}</v-list-item-title>
+                    </v-list-item-content>
+                  </template>
+                  <v-list-item v-for="{ id, name } in tags" :key="id" @click.prevent="fetchBookmarks">
+                    <v-list-item-content>
+                      {{ name }}
+                    </v-list-item-content>
```

Comment:
```suggestion
                    <v-list-item-content>
                      <v-list-item-title class="font-weight-black text-h7">
                      {{ title }}
                      </v-list-item-title>
                    </v-list-item-content>
                  </template>
                  <v-list-item v-for="{ id, name } in tags" :key="id" @click.prevent="fetchBookmarks">
                    <v-list-item-content>
                      {{ name }}
                    </v-list-item-content>
```

이렇게 하면 안되는건가요?? 순수한 궁금증입니다..

Reply:
줄수가 넘지 않아서 prettier가 안내려준거 같아요 ㅎㅎ 내리는게 더 깔끔하네요!